pkg/controllers/subnetset/subnetset_controller.go

```diff
@@ -69,12 +70,19 @@ func (r *SubnetSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 				vpcNetworkConfig := r.VPCService.GetVPCNetworkConfigByNamespace(obj.Namespace)
 				if vpcNetworkConfig == nil {
 					err := fmt.Errorf("failed to find VPCNetworkConfig for namespace %s", obj.Namespace)
-					log.Error(err, "operate failed, would retry exponentially", "subnet", req.NamespacedName)
+					log.Error(err, "operate failed, would retry exponentially", "subnetset", req.NamespacedName)
 					updateFail(r, ctx, obj, "")
 					return ResultRequeue, err
 				}
 				obj.Spec.IPv4SubnetSize = vpcNetworkConfig.DefaultSubnetSize
 			}
+			if util.IsPowerOfTwo(obj.Spec.IPv4SubnetSize) == false {
+				errorMsg := fmt.Sprintf("ipv4SubnetSize has invalied size %d,  which needs to be >= 16 and power of 2", obj.Spec.IPv4SubnetSize)
+				log.Error(nil, "ipv4SubnetSize has invalied size", "subnetset", req.NamespacedName)
```

Comment:
```suggestion
				log.Error(nil, errorMsg, "subnetset", req.NamespacedName)
```

Reply:
Merged the error message with the log to reduce redundancy and improve clarity.

Reply:
Thanks for advice, fixed.